charts/generic-service/templates/batch-cronjob.yaml

```
@@ -0,0 +1,46 @@
+{{- /*
+This template is a work in progress - please do not use yet!
+*/}}
+---
+{{- range $job := .Values.batchjobs }}
+{{- with $ -}}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "generic-service.fullname" . }}-{{ $job.name }}
+  labels:
+    {{- include "generic-service.labels" . | nindent 4 }}
+spec:
```

Could also add this maybe for more control:

```suggestion
spec:
  suspend:  {{ $job.suspend | default False }}
```